packages/react/src/composite/Accordion/CHANGELOG.md

```
@@ -2,6 +2,11 @@
 
 The Accordion component allows us to compress content into containers that can be opened or closed.
 
+## 2026-01-19 `1.7.0`
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)